From 2c21c2a321ee3515923d7e142141cd010137d420 Mon Sep 17 00:00:00 2001
From: Quentin Schulz <quentin.schulz@bootlin.com>
Date: Wed, 25 Jul 2018 15:27:54 +0200
Subject: [PATCH 2/4] do not use base_read_file

It's been deprecated for many releases and was removed in sumo, let's use
oe.utils.read_file instead.

Signed-off-by: Quentin Schulz <quentin.schulz@bootlin.com>
---
 recipes-ti/dsplink/ti-dsplink.inc                         | 2 +-
 recipes-ti/local-power-manager/ti-local-power-manager.inc | 2 +-
 recipes-ti/syslink/ti-syslink.inc                         | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/recipes-ti/dsplink/ti-dsplink.inc b/recipes-ti/dsplink/ti-dsplink.inc
index e0096155c770..606b9e5c6975 100644
--- a/recipes-ti/dsplink/ti-dsplink.inc
+++ b/recipes-ti/dsplink/ti-dsplink.inc
@@ -150,7 +150,7 @@ do_compile() {
       all
 }
 
-KERNEL_VERSION = "${@base_read_file('${STAGING_KERNEL_DIR}/kernel-abiversion')}"
+KERNEL_VERSION = "${@oe.utils.read_file('${STAGING_KERNEL_DIR}/kernel-abiversion')}"
 
 do_install () {
 	
diff --git a/recipes-ti/local-power-manager/ti-local-power-manager.inc b/recipes-ti/local-power-manager/ti-local-power-manager.inc
index 0e5d4f297a47..5321251a2709 100644
--- a/recipes-ti/local-power-manager/ti-local-power-manager.inc
+++ b/recipes-ti/local-power-manager/ti-local-power-manager.inc
@@ -92,7 +92,7 @@ do_compile () {
     XDCPATH=${S} ${XDC_INSTALL_DIR}/xdc -PR .
 }
 
-KERNEL_VERSION = "${@base_read_file('${STAGING_KERNEL_DIR}/kernel-abiversion')}"
+KERNEL_VERSION = "${@oe.utils.read_file('${STAGING_KERNEL_DIR}/kernel-abiversion')}"
 
 do_install () {
 
diff --git a/recipes-ti/syslink/ti-syslink.inc b/recipes-ti/syslink/ti-syslink.inc
index 5ff6af7c45cd..0968a986f102 100644
--- a/recipes-ti/syslink/ti-syslink.inc
+++ b/recipes-ti/syslink/ti-syslink.inc
@@ -181,7 +181,7 @@ do_compile() {
 	 `${XDC_INSTALL_DIR}/bin/xdcpkg ${SYSLINK_ROOT}/ti/syslink | grep samples`
 }
 
-KERNEL_VERSION = "${@base_read_file('${STAGING_KERNEL_DIR}/kernel-abiversion')}"
+KERNEL_VERSION = "${@oe.utils.read_file('${STAGING_KERNEL_DIR}/kernel-abiversion')}"
 
 do_install () {
 
-- 
2.7.4

